configuration/cdap-site.xml

```diff
@@ -1047,6 +1082,139 @@
     </value-attributes>
   </property>
 
+  <!-- Messaging System Configuration -->
+
+  <property>
+    <name>messaging.container.instances</name>
+    <value>1</value>
+    <description>
+      Number of instances for the messaging service
+    </description>
+    <final>true</final>
+  </property>
+
+  <property>
+    <name>messaging.container.memory.mb</name>
+    <value>${master.service.memory.mb}</value>
+    <description>
+      Memory in megabytes for each messaging service instance
+    </description>
+  </property>
+
+  <property>
+    <name>messaging.container.num.cores</name>
+    <value>${master.service.num.cores}</value>
+    <description>
+      Number of virtual cores for each messaging service instance
+    </description>
```

Comment:
Copy the elements from `log.saver.container.num.cores` for `value-attributes` and sub-elements